systemd/fedmsg-verify-missing.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Fedmsg migration: ZeroMQ to AMQP
+After=network.target
+Documentation=https://github.com/fedora-infra/fedmsg-migration-tools
+
+[Service]
+Type=simple
+EnvironmentFile=/etc/sysconfig/fedmsg-migration-tools
+ExecStart=/usr/bin/fedmsg-migration-tools verify_missing
+#ExecStart=/usr/bin/twistd-3 -n --pidfile= fedmsg-verify-missing
```

Comment:
I recommend dropping the commented-out version and we can change it if/when we run things with twistd

Reply:
Agreed.